rustls/src/lib.rs

```diff
@@ -271,6 +271,19 @@
 //!
 //!   Note that aws-lc-rs has additional build-time dependencies like cmake.
 //!   See [the documentation](https://aws.github.io/aws-lc-rs/requirements/index.html) for details.
+//!
+//!
+//! # Unexpected EOF
+//!
+//! TLS has a [`close_notify` mechanism to prevent truncation attacks](https://datatracker.ietf.org/doc/html/rfc8446#section-6.1).
+//! According to the TLS RFCs, each party is required to send it before closing the write side of the connection.
```

Comment:
Above you talk about the close notify mechanism generally, but I think it means the "it" in the subsequent sentences isn't very clear. WDYT about saying "an alert" instead of "it":

```suggestion
//! According to the TLS RFCs, each party is required to send an alert before closing the write side of the connection.
```